src/main/java/org/betonquest/betonquest/api/profile/UUIDProfileProvider.java

```diff
@@ -12,21 +10,17 @@
 import java.util.UUID;
 
 /**
- * Converts the player to the Profile
+ * Default implementation for {@link ProfileProvider}.
```

Comment:
As this is again a like temporary work around, until we finally implementing profiles fully working, This should not be in the API. Move this implementation to a new package `profile` in the root package.

Reply:
Moving it outside